### PR TITLE
fix: Release syndesis-operator.yml with the proper version

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: tools/bin/commands/release [hunk 1]
```diff
@@ -290,9 +307,14 @@ docker_push() {
     echo "==== Pushing to Docker Hub"
     for module in $ALL_IMAGE_MODULES; do
         local image="syndesis/syndesis-$module"
-        docker tag "$image:$release_version" "$image:$moving_tag"
         docker push "$image:$release_version"
```
[the remark this answers is not in the document]
Comment (rhuss) [reply tools/bin/commands/release]: See the change below these lines, actually, we are creating "moving tags" for all images except the operator image.

The reason is, that for all except the operator images like 1.4.2 and 1.4 (the moving tag, always pointing to the latest on this line) will be identical.

However, the operator image with the moving tag must also contain the template with the moving tag (i.e. referencing images like :1.4). So a separate build and push are required. This happens in the function `create_moving_tag_release`.

**Comment by rhuss** [comment]
> I think we won't version-tag images other than operator image with this change, am I wrong?

Not sure if I understand you correctly, but it's that we have to generate a special Docker image for the moving tag. We always release the full image tag (1.4.2) and a moving tag (1.4) so that with the trigger we can benefit from automatic updates (assuming that patch level updates don't require an upgrade process).

**Comment by heiko-braun** [comment]
@rhuss can you create a blocking issue for this? Just for the records ...

**Comment by rhuss** [comment]
I think I'm good to merge to master. Let's try a 1.5.1 release there and then I will create a backport to 1.4.x and create a 1.4.3 release from there.

**Pull request merged**